scripts/analytics/benchmarks.py

```diff
@@ -217,15 +209,15 @@ def main():
     projects_forked = get_projects_forked()
     projects_registered = get_projects_registered()
 
-    number_projects = len(projects)
+    number_projects = projects.count()
```

Comment:
You're actually pretty close to the truth- ODM  already treats `len` and `count` the same way in ODM- unless you call `list` on something first. (this is an ODM behavior rather than python internals per se)

In this case, I'm deliberately being stubborn and using the queryset method, specifically to discourage us from treating lists and querysets as interchangeable. (given our problems in the past, I'm trying to make *sure* that this exact syntax breaks if given a list)